<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lab1Window</class>
 <widget class="QMainWindow" name="lab1Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>190</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Лаба 1 - передача файлов</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="endpointTypeTab">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>391</width>
      <height>141</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="TransmitTab">
     <attribute name="title">
      <string>Передача</string>
     </attribute>
     <widget class="QLineEdit" name="ipAddrEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="inputMask">
       <string>000.000.000.000;_</string>
      </property>
      <property name="text">
       <string>127.000.000.001</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="portEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="text">
       <string>5915</string>
      </property>
      <property name="maxLength">
       <number>7</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="protocolSelect">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IP-адрес:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Порт:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Протокол:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ReceiveTab">
     <attribute name="title">
      <string>Приём</string>
     </attribute>
     <widget class="QLineEdit" name="listenPortEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5915</string>
      </property>
      <property name="maxLength">
       <number>7</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Слушать порт:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="listenProtocol">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Протокол:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startListenBtn">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Слушать</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="udpListenAdress">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>83</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="inputMask">
       <string>000.000.000.000;_</string>
      </property>
      <property name="text">
       <string>127.000.000.001</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>44</x>
        <y>80</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>IP-адрес:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="cancelBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>154</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="lab1.qrc"/>
 </resources>
 <connections/>
</ui>
